src/component/Label.js

```diff
@@ -297,7 +298,7 @@ function Label(props) {
 
   return (
     <Text
-      className={"recharts-label" + (className ? ' ' + className : '')}
+      className={`recharts-label ${className}`}
```

Comment:
@akram-rameez Use the util function of `classnames`, please.
```jsx
import classNames from 'classnames';
<Text className={classNames('recharts-label', className);}>
```

Reply:
alright 👍